modules/navier_stokes/src/fvkernels/INSFVMomentumAdvection.C

```diff
@@ -436,15 +436,8 @@ INSFVMomentumAdvection::interpolate(Moose::FV::InterpMethod m,
                                     const ADRealVectorValue & elem_v,
                                     const ADRealVectorValue & neighbor_v)
 {
-  auto tup = Moose::FV::determineElemOneAndTwo(*_face_info, *_p_var);
-  const Elem * const elem_one = std::get<0>(tup);
-  const Elem * const elem_two = std::get<1>(tup);
-  const bool elem_is_elem_one = std::get<2>(tup);
-  mooseAssert(elem_is_elem_one
-                  ? elem_one == &_face_info->elem() && elem_two == _face_info->neighborPtr()
-                  : elem_one == _face_info->neighborPtr() && elem_two == &_face_info->elem(),
-              "The determineElemOneAndTwo utility determined the wrong value for elem_is_elem_one");
-  mooseAssert(elem_is_elem_one, "FVFluxKernel elem should coincide with the faceInfo elem.");
+  const Elem * const elem = &_face_info->elem();
+  const Elem * const neighbor = _face_info->neighborPtr();
```

Comment:
This is actually really good. Now when people do `git blame` you will own all these lines and they will think you wrote bugs instead of me :laughing:

Reply:
jaja. Well I ll point them to the reviewer who merged it